environment.yml

```diff
@@ -24,15 +24,13 @@ dependencies:
 - pylint
 - coverage
 - requests
-- xlwt
-- statsmodels
-- linearmodels
-- rpy2
 - black
 - pre-commit
+- taxcalc
 - pip:
   - jupyter-book>=0.8.0
-  - cs-kit
-  - cs2tc
-  - pandas-datareader
   - git+https://github.com/PSLmodels/OG-USA.git
+  - git+https://github.com/nikhilwoodruff/openfisca-core.git
```

Comment:
@jdebacker I think this is causing the issue - the openfisca-uk requirements require:
```yaml
- git+https://github.com/nikhilwoodruff/openfisca-core
```
rather than with the ```.git```. It might be easier just to drop this requirement for the sake of clarity and let it be installed as an openfisca-uk dependency?